cmd/notation/verify.go

```diff
@@ -76,7 +84,9 @@ func runVerify(command *cobra.Command, opts *verifyOpts) error {
 
 	// resolve the given reference and set the digest
 	ref, err := resolveReference(command.Context(), &opts.SecureFlagOpts, reference, sigRepo, func(ref registry.Reference, manifestDesc ocispec.Descriptor) {
-		fmt.Fprintf(os.Stderr, "Warning: Always verify the artifact using digest(@sha256:...) rather than a tag(:%s) because resolved digest may not point to the same signed artifact, as tags are mutable.\n", ref.Reference)
+		if opts.outputFormat == ioutil.OutputPlaintext {
```

Comment:
Do we need to do this if warnings are printed to stderr?

Reply:
i guess it's not required if customers just need to read from stdout, verified this locally w/
`./bin/notation verify $IMAGE --output json >> stdout.txt 2> stderr.txt`